### ISSUE TITLE
approving an application creates an extra copy

### ISSUE THREAD

**Issue by TravisWalterLBNL** [LBNL-ETA/BEDES-Manager#218]
I created a new application and exported some app terms and composite terms from a .csv file using my regular account. I logged out and logged back in as an admin user, then went into the application options for the newly created account and changed Sharing to Approved. This caused the application to show up twice in the list of applications, and each of the app terms to show up twice in the list of app terms. The app and app terms should each only show up once in their respective lists.

**Comment by wizonesolutions** [comment]
Ah, yes, I have encountered this one myself. It's actually only one application in the database (if you delete one, I believe both disappear), but I think something in one of the queries is confounding things. It probably happened when we implemented the Approved status, then implement the "show public" checkbox.

In technical terms, the database query is probably resulting in a cross multiply during a join. I have a simple workaround in mind that I will have to check the feasibility of in Postgres.

**Comment by wizonesolutions** [comment]
Yep, the issue was exactly as I thought. I've make a fast fix for this issue to unblock us. It's deploying to staging, so check in 15-20 mins.

Then, we should open a follow-up to fix it properly. Basically, when approving an application, the administrator is given an Owner role. I'm not actually sure why this happens, since they have access anyway, and it's not necessary. A non-admin owner can't un-approve an application (not even an admin can). And we show BEDES as the owner whenever the scope is Approved, so it's not important for that either.

But it's basically a cleanup task and probably low priority. It is not significant technical debt at this stage.

**Comment by TravisWalterLBNL** [comment]
OK, the duplicates are gone on staging. I'll keep this open to remind us that it needs a permanent fix, but I'll change it to low priority.
